<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameWidget</class>
 <widget class="QWidget" name="RenameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_upload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Суффикс</string>
   </property>
   <widget class="QRadioButton" name="radioButton_1Typ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1тур</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2Typ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2тур</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_without">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Без суффикса</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_other">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Другой</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_suffix">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Имена участников</string>
   </property>
   <widget class="QTextEdit" name="textEdit_participant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Протокол</string>
   </property>
   <widget class="QPushButton" name="pushButton_protocol">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_protocol">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_rename">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Переименовать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Папка для переименованных работ</string>
   </property>
   <widget class="QPushButton" name="pushButton_distDir">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Папка безымянных работ</string>
   </property>
   <widget class="QPushButton" name="pushButton_namelessDir">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_namelessDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Классы участников</string>
   </property>
   <widget class="QTextEdit" name="textEdit_grades">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
